### PR TITLE
Use the new AgentMajorVersion parameter in the e2e tests

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Use the new AgentMajorVersion parameter in the e2e tests

### Motivation

https://github.com/DataDog/test-infra-definitions/pull/1192

### Additional Notes

Relates to https://github.com/DataDog/test-infra-definitions/pull/1192

https://datadoghq.atlassian.net/browse/ADXT-680

### CODE REVIEW

File: test/new-e2e/pkg/runner/configmap.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ const (
 	AgentAPPKey = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentAPPKeyParamName
 	// AgentPipelineID pulumi config parameter name
 	AgentPipelineID = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentPipelineID
+	// AgentMajorVersion pulumi config parameter name
+	AgentMajorVersion = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentMajorVersion
```
Comment (f4usto) [inline test/new-e2e/pkg/runner/configmap.go:30]: **thought**:
Should we already extend for minor and point versions?

**Review by f4usto — APPROVED** [review]
lgtm, placed only a question

File: test/new-e2e/pkg/runner/configmap.go (hunk 1 above)
Comment (FlorentClarret) [reply test/new-e2e/pkg/runner/configmap.go:30]: That's actually not really needed, at least for now. This parameter is currently used to switch from one "bucket" to the other (we have one for a6 and one for a7). Until now, the a7 bucket was hardcoded si we were not able to pull the a6 artefacts. This is now possible

**Review by pducolin — APPROVED** [review]

**Pull request merged**